### ISSUE TITLE
Cannot reproduce pretrained VAEformer

### ISSUE THREAD

**Issue by tung-nd** [taohan10200/CRA5#9]
Hi. I used your exact model architecture and trained the VAEformer model but could not reproduce your results (first row of Table 1). Specifically, I was able to get an overall MSE of 0.004, which is lower than your MSE, but my variable-specific RMSE was worse, especially for Z500 and T2m. Did you use a weighted loss function to upweigh the importance of these variables during training? Thank you!

**Comment by taohan10200** [comment]
Yes, we use an uncertainty loss to automatically weight different variables so that make them can achieve better optimization in a systematic view.

**Comment by tung-nd** [comment]
Thank you for your response. Do you plan to release the code the reproduces that?

**Comment by taohan10200** [comment]
We still try to polish the paper for publication. The training code would be released upon the acceptance.
